events_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 
 	// this is *C comes from
+	"github.com/PagerDuty/godspeed"
```

Comment:
The way this import is being added is messing with the comment above. Might be best to put the import above this comment.

Also, I've done gnarly typos in here, huh?

Reply:
Oops didn't see that. I'll fix it, it was just the auto goimports formatting. Typos are always fun! ;-)